### ISSUE TITLE
Start from User App Data doesn't handle Unicode input

### ISSUE THREAD

**Issue by ekuzneczova1988** [quasar/QuasarRAT#410]
The problem is when you start from User App Data and name of windows user has Unicode:
![default](https://cloud.githubusercontent.com/assets/15610263/10990283/8e18a4c8-847b-11e5-98cd-37c104194be8.png)

**Comment by yankejustin** [comment]
Thanks for the report.

**Comment by d3agle** [comment]
This is the error message translates to "Windows cannot find ... make sure you type the name correctly, and then try again."

It's likely a registry error and not related to this program (corruption, invalid path, reference to an orphaned entry).

**Comment by MaxXor** [comment]
I can't reproduce this with the username "Колмогоров". Autostart is also working fine.
[Screenshot](https://i.imgur.com/7WuE5PK.png)

**Comment by d3agle** [comment]
I haven't got around to testing the latest commits, but there are likely more bugs in this area.

When I tested a few days ago, I used a username with Russian characters, as well as a Russian internal display language.  When I tried using the password recovery, the client process crashed.  Something with the nss3.dll module.  I won't have time to test this until sometime tomorrow, but I wanted to point it out.
